app/src/test/java/org/oppia/android/app/player/state/StateFragmentLocalTest.kt

```diff
@@ -1344,6 +1338,7 @@ class StateFragmentLocalTest {
     appStringIetfTag = "ar",
     appStringAndroidLanguageId = "ar"
   )
+  @Ignore // TODO(#4492): Update the arabic string for show hint button.
```

Comment:
Reminder to fix (which I think can be if translations are restored).

Reply:
Removed as translations have been restored.